### PR TITLE
Model set up

### PR DESCRIPTION
**Pull request by Obehi234**
Set up model and created corresponding dependencies for network call.

### CODE REVIEW

File: app/src/main/java/com/example/randomuserapplication/network/UserService.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.example.randomuserapplication.network
+
+import com.example.randomuserapplication.model.UserList
+import com.example.randomuserapplication.util.Constants.END_POINT
+import retrofit2.Response
+import retrofit2.http.GET
+
+interface UserService {
+    @GET(END_POINT)
```
Comment (IfyUnoka4) [inline app/src/main/java/com/example/randomuserapplication/network/UserService.kt]: write out the endpoint here. don't extract it

File: app/src/main/java/com/example/randomuserapplication/util/Constants.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.example.randomuserapplication.util
+
+object Constants {
+    const val BASE_URL = "https://randomuser.me/"
+    const val END_POINT = "api/?results=500&inc=name,picture,location,nat,gender,dob,login&noinfo"
+}
```
Comment (IfyUnoka4) [inline app/src/main/java/com/example/randomuserapplication/util/Constants.kt:5]: why is your endpoint in this form? why is this `inc=name,picture,location,nat,gender,dob,login&noinfo` needed?

File: app/src/main/java/com/example/randomuserapplication/network/UserService.kt (hunk 1 above)
Comment (Obehi234) [reply app/src/main/java/com/example/randomuserapplication/network/UserService.kt]: Oh ok

File: app/src/main/java/com/example/randomuserapplication/util/Constants.kt (hunk 2 above)
Comment (Obehi234) [reply app/src/main/java/com/example/randomuserapplication/util/Constants.kt:5]: In the documentation, they said if you wanted specific information, do it like this. You want me to change it? To extract all the information?
api

**Review by IfyUnoka4 — APPROVED** [review]

**Pull request merged**